backend/libraries/download.py

```diff
@@ -56,9 +133,11 @@ def get_library(
     library_dir.mkdir(parents=True, exist_ok=True)
 
     if "git" in download_info:
-        download_git(
+        res = download_git(
             library_name, library_version, library_dir, download_info["git"], force
         )
+        if res is False:
+            return False
```

Comment:
isn't this going to happen anyway when we return res at the end? or is there some nuance between none and false I'm missing

Reply:
Nuance is meant to be `None` means it wasn't updated, whereas False means it failed

Reply:
ahh I misread the indentation level, nm!